scipy/spatial/distance.py

```diff
@@ -931,6 +933,57 @@ def kulsinski(u, v, w=None):
     return (ntf + nft - ntt + n) / (ntf + nft + n)
 
 
+def kulczynski1(u, v, w=None):
```

Comment:
Make extra parameters keywords only.
```suggestion
def kulczynski1(u, v, *, w=None):
```

Reply:
Other functions like, `hamming`, `jaccard`, `kulsinski`, etc, follow (u, v, w=None) signature in this file. Should I also make them, `(u, v, *, w=None)`.

Reply:
We cannot change previous functions without breaking backward compatibility here. So just do it for new functions or when adding new parameters.